src/components/dialogs/PostInteractionSettingsDialog.tsx

```diff
@@ -257,6 +257,11 @@ export function PostInteractionSettingsForm({
       newSelected.splice(i, 1)
     }
 
+    // if nothing is selected, default to 'everybody'
+    if (newSelected.length === 0) {
+      newSelected.push({type: 'everybody'})
+    }
```

Comment:
Since newSelected.length === 0 is a consequence of  the logic starting above, at :253, imo it makes more sense to write this more inclusively with that block alike 

```
  /**
   * check if the incoming setting DNE -> add to the list
   * is the only setting + being removed -> revert to default
   * or is being removed from the others -> splice
   */
  const i = newSelected.findIndex(v => isEqual(v, setting));
  if (i === -1) {
    newSelected.push(setting);
  } else if (newSelected.length === 1) {
    newSelected = [{type: 'everybody'}];
  } else {
    newSelected.splice(i, 1);
  }
```

Reply:
Good point! Updated :) 
Just slightly modified your comments to expand abbreviations